lms/envs/common.py

```diff
@@ -3063,6 +3063,14 @@ def _make_locale_paths(settings):
         "account_privacy",
         "accomplishments_shared",
         "extended_profile",
+        "position",
+        'first_name',
```

Comment:
Please pay attention to the quotation it is a good practice to use similar quotes/double quotes in the same block.

Reply:
Fixed